### PR TITLE
Global config for GitHub token

### PR DESCRIPTION
**Pull request by Soullivaneuh**
See https://github.com/composer/composer/issues/4282#issuecomment-123662454.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@ matrix:
 before_script:
   - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - composer selfupdate
+  - composer config -q -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
   - composer config -q github-oauth.github.com $GITHUB_OAUTH_TOKEN
```
Comment (OskarStark) [inline .travis.yml]: do we need both lines?

Comment (Soullivaneuh) [inline .travis.yml]: Yes. See comment on PR subject.

Comment (stof) [inline .travis.yml]: no you don't. ``composer config -g`` configures system-wide (well, for this user, not for the whole system). It is not the same than ``composer global config``, which would just change the config of the "global" project (``composer global`` is just changing the working directory to a different one and run composer commands on this special project)

Comment (Soullivaneuh) [inline .travis.yml]: @stof Oh ok, misunderstood. So I just need the `-g` one?

Comment (stof) [inline .travis.yml]: normally, yes

Comment (Soullivaneuh) [inline .travis.yml]: Will try it, thanks.

Comment (OskarStark) [inline .travis.yml]: :+1:

**Pull request merged**